ui/components/ui/box/README.mdx

```diff
@@ -134,14 +135,10 @@ import Box from '../ui/box';
 
 Use the `backgroundColor` prop along with the `COLORS` object from `ui/helpers/constants/design-system.js` to change background color. The `backgroundColor` prop accepts [responsive props](#responsive-props) in the form of array props.
 
-````jsx
```

Comment:
This fixes a broken story

## Before

![](https://user-images.githubusercontent.com/8112138/183728497-012d58d3-c178-46cf-ac98-0c7e7a2de5ba.png)

## After

<img width="1440" alt="Screen Shot 2022-08-09 at 2 55 33 PM" src="https://user-images.githubusercontent.com/8112138/183768800-661031a6-f9bc-444c-8129-ae412d9ce56e.png">